ampd/src/tofnd/mod.rs

```diff
@@ -7,9 +7,10 @@ use crate::url::Url;
 
 pub mod client;
 pub mod error;
+pub mod grpc;
```

Comment:
Why is this public?

Reply:
the lib needs the generated client to connect